cf/api/services.go

```diff
@@ -255,6 +256,22 @@ func (repo CloudControllerServiceRepository) FindServicePlanByDescription(planDe
 	return planGuid, apiErr
 }
 
+func (repo CloudControllerServiceRepository) ListServicesFromManyBrokers(brokerGuids []string) (services []models.ServiceOffering, err error) {
```

Comment:
Yeah, it's another one of those things we're trying to change by changing it wherever we see it. Sorry for these sorts of hidden trip ups.